charts/paperless/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+appVersion: 2.7.0
+description: Paperless - Index and archive all of your scanned paper documents
+name: paperless
+version: 0.1.0
+keywords:
+  - paperless
+home: https://github.com/k8s-at-home/charts/tree/master/charts/paperless
+icon: https://raw.githubusercontent.com/the-paperless-project/paperless/master/src/paperless/static/paperless/img/logo-dark.png
+sources:
+  - https://github.com/the-paperless-project/paperless
+maintainers:
+  - name:
```

Comment:
Please add the maintainers info. Feel free to add the default maintainers if you don't wish to add yourself.